docker/tasks/main.yml

```diff
@@ -27,8 +26,12 @@
 - name: Docker repository
   become: yes
   ansible.builtin.apt_repository:
-    repo: deb [arch=amd64 signed-by=/usr/share/keyrings/docker-archive-keyring.gpg] https://download.docker.com/linux/ubuntu {{ ansible_distribution_release }} stable
-  when: "ansible_architecture == 'arm64'"
+    repo: deb [arch=arm64 signed-by=/usr/share/keyrings/docker-archive-keyring.gpg] https://download.docker.com/linux/ubuntu {{ ansible_distribution_release }} stable
+  when: "ansible_architecture == 'aarch64'"
+
+- name: Run the equivalent of "apt-get update"
```

Comment:
Can you just combine this to the job below like you had it, but just remove the cache_valid_time - no need to have this as a separate task when its an option in the other install step

Reply:
Changed these steps